mitodl/ocw-hugo-themes
# fix: images scaled

#### Pre-Flight checklist


#### What are the relevant tickets?
Closes https://github.com/mitodl/ocw-hugo-themes/issues/65

#### What's this PR do?
- Gives 100% width to all the images in instructor insights only. 

#### How should this be manually tested?
- Please also go through this [comment](https://github.com/mitodl/ocw-hugo-themes/issues/65#issuecomment-1022215546)
- Open any course
- Go to instructor insights and verify that the images are scaled and takes 100% width of the section.
- Since a few changes have been made in hugo so please also verify that all the other sections/pages are working fine and images of those pages are **not** scaled.
- Repeat the above step for: 
    - Multiple screen dimensions
    - Multiple browsers  

#### Screenshots (if appropriate)
![image](https://user-images.githubusercontent.com/93309234/151172904-d6d32fc3-33f5-426b-befb-6e39445adc19.png)

![image](https://user-images.githubusercontent.com/93309234/151172938-72b97397-883f-41f7-b53e-b74aaee475ed.png)
